modules/environment/aws/code-services/main.tf

```diff
@@ -202,3 +202,52 @@ resource "aws_sqs_queue_policy" "code_services_queue_policy" {
 }
 POLICY
 }
+
+resource "aws_iam_role" "sqs_role" {
```

Comment:
this role / policy is for the event mapper, right?  if so, can you make these names a bit more specific?  this resource, its name, and the `aws_iam_role_policy` resource and its name should probably be changed